avocado/core/job.py

```diff
@@ -488,6 +479,23 @@ def _run(self):
 
         return self.exitcode
 
+    def create_test_suite(self):
+        """
+        Creates the test suite for this Job
+
+        This is a public Job API as part of the documented Job phases
```

Comment:
Yes, I know, I was just stating my opinion about the docstring and added a suggestion (more-like brainstorm). As for the import into `avocado` namespace, yes, that's probably the way when we're ready to support it, although that means importing the whole `Job`, not just the methods, so private/x_ prefixes is something we'll have to consider first.

Anyway right now this is fine (I'd just fine-tuned the docstring, if possible, for example use `:note: future public api` or something more machine-readable :smile: - by machine I don't really mean machine, but more visible in docstrings).